luky-borg-backup

```diff
@@ -7,6 +7,21 @@ if [ -z "$REPOSITORY" ]; then
     exit
 fi
 
+notify() {
+    echo $(echo $1|tr '[:lower:]' '[:upper:]'): $2
+    if [ $(command -v sudo) -a $(command -v notify-send) ]; then
```

Comment:
testing this on a machine without notify-send gives me the following error:
```
-bash: [: /usr/bin/sudo: unary operator expected
```
Which is because only sudo could be found and `$(command -v notify-send)` returns an empty string. Test `/usr/bin/sudo -a ` is invalid then because it should have been given a second argument.
This can be solved by adding double quotes to the parameters like `"$(command -v sudo)" -a "$(command -v notify-send)"` which would result in `"/usr/bin/sudo" -a ""` in my case, which tests as false.

So, please add double quotes like in `if [ "$(command -v sudo)" -a "$(command -v notify-send)" ]; then`